cms/static/js/certificates/spec/views/certificate_editor_spec.js

```diff
@@ -90,6 +90,16 @@ function(_, Course, CertificateModel, SignatoryModel, CertificatesCollection, Ce
         uploadDialogTpl = readFixtures('upload-dialog.underscore');
 
         beforeEach(function() {
+            window.course = new Course({
```

Comment:
That wasn't clear. I meant that this setup is repeated in three different spec files, and it seems that others might need it in the future. In fact I'm fine with leaving it as is. I'd rather that our views didn't depend upon global variables like this anyway... the information should be passed in to the factory IMO.

There is a repeating of this code at line 105 below here. Is this the bad merge you were referring to? That should be cleaned up.